calculators/handlers/Prestige.js

```diff
@@ -24,7 +23,7 @@ class PrestigeHandler {
         if (prices[item.itemId]) return;
         const prestige = prestiges[item.itemId];
         for (const prestigeItem of prestige) {
-            const foundItem = getHypixelItemInformationFromId(prestigeItem);
+            const foundItem = item?.skyblockItem;
```

Comment:
This should be reverted. This is getting the item information for a different item than is being calculating. `prestigeItem !== item.itemId`

Reply:
oops didn't see